README.md

```diff
@@ -26,11 +26,9 @@ Adb should be enabled on your instance.
 
 * Run this docker
 
-`docker run --net=host -P -v <path_to_genymotion_cloud_ssl_tool>:/opt/work/ --env package=true --env keystorepassword=<kspassword> --env certpassword=<kppassword> -t -i <name> /opt/work/package.sh <relpath_to_privkey.pem> <relpath_to_fullchain.pem>`
+`docker run -v <full_path_to_genymotion_cloud_ssl_tool>:/opt/work/ --env package=true --env keystorepassword=<kspassword> --env certpassword=<kppassword> --env domain=<domainName> -t -i <name>
```

Comment:
` is missing at the end of the docker run command

Also, detail, but I find the synthax with ``` blocks more readable for README commands